engine/src/java/com/runrev/android/Engine.java

```diff
@@ -2232,9 +2232,7 @@ else if (resultCode == Activity.RESULT_OK)
 					float t_width = t_bitmap.getWidth();
 					float t_height = t_bitmap.getHeight();
 					
-					InputStream t_exif_in = ((LiveCodeActivity)getContext()).getContentResolver().openInputStream(t_photo_uri);
-					
-					ExifInterface t_exif = new ExifInterface(t_exif_in);
+					ExifInterface t_exif = new ExifInterface(t_photo_uri.getPath());
```

Comment:
I'm a little concerned that this will break on newer Android versions. Android versions >= 7 (IIRC - API Level 24 by any chance - that's where the new constructor was introduced) are more strict about how apps access files.

In this case the input could be either a Uri from picking from a library, or presumably a temp file from taking a photo.

Can you confirm that all aspects of mobilePickPhoto still work with this change in both Andoid6 and 7+?